web/src/title/Loading.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  // TODO Upstream to svelte-utils
+
+  // Block the rest of the page while 'loading' is non-empty
+
+  export let loading: string[];
+
+  let inner: HTMLDivElement;
+
+  $: if (loading.length > 0 && inner) {
```

Comment:
This seems to happen a little late / misses the last message sometimes. Searches show some CSS approaches I'll play with eventually